src/stories/LineChart.stories.tsx

```diff
@@ -180,3 +180,348 @@ export const MultiLineChartWithFilledBackGround: Story = {
+          borderColor: "#767676",
+          fill: true,
+        },
+        {
+          data: [40, 40, 25, 55, 80, 35, 80, 55, 20, 20],
+          borderColor: "#bebebe",
+          fill: true,
+        },
+      ],
+    },
+    className:
+      "bg-white-500 border-2 border-solid border-[#ededed] px-6 pt-2 pb-0 rounded-xl w-[46rem] h-96",
+  },
+};
+
+export const MultiLineCurvyChartWithMultipleLines: Story = {
+  args: {
+    options: {
+      responsive: true,
+      plugins: {
+        legend: {
+          display: false,
+        },
+        title: {
+          display: true,
+          text: "CPU Usage",
+          position: "top",
+          align: "start",
+          font: {
+            size: 18,
+          },
+          padding: {
+            top: 20,
+            bottom: 24,
+          },
+        },
+      },
+      scales: {
+        x: {
+          grid: {
+            display: false,
+          },
+          border: {
+            color: "#eaeaea",
+            width: 2,
+          },
+          title: {
+            display: true,
+            text: "Time Intervals",
+            font: {
+              size: 12,
+              weight: 600,
+            },
+            padding: {
+              top: 12,
+              bottom: 0,
+            },
+          },
+          ticks: {
+            padding: 2,
+          },
+        },
+        y: {
+          display: true,
+          beginAtZero: true,
+          max: 100,
+          stepSize: 6,
+          grid: {
+            drawTicks: true,
+            tickColor: "#ffffff",
+          },
+          ticks: {
+            display: true,
+            stepSize: 20,
+            count: 6,
+            callback: function (value) {
+              return `${value}%`;
+            },
+          },
+          border: {
+            dash: [5, 10],
+            color: "#ffffff",
+          },
+        },
+      },
+      datasets: {
+        line: {
+          pointStyle: false,
+          borderWidth: 3,
+          tension: 0.4,
+        },
+      },
+    },
+    data: {
+      labels: [
+        "10:01",
+        "10:02",
+        "10:03",
+        "10:04",
+        "10:05",
+        "10:06",
+        "10:07",
+        "10:08",
+        "10:09",
+        "10:10",
+      ],
+      datasets: [
+        {
+          data: [20, 20, 40, 60, 35, 60, 40, 40, 22, 20],
+          borderColor: "#767676",
+        },
+        {
+          data: [40, 40, 25, 55, 80, 35, 80, 55, 20, 20],
+          borderColor: "#bebebe",
+        },
+        {
+          data: [20, 20, 40, 40, 80, 70, 30, 30, 20, 20],
+          borderColor: "#b2b2b2",
+        },
+        {
+          data: [40, 40, 60, 40, 70, 60, 70, 30, 40, 30],
+          borderColor: "#ededed",
+        },
+      ],
+    },
+    className:
+      "bg-white-500 border-2 border-solid border-[#ededed] px-6 pt-2 pb-0 rounded-xl w-[46rem] h-96",
+  },
+};
+
+export const MultiLineDarkBgCurvyLineChart: Story = {
```

Comment:
`MultiLineDarkBackGroundCurvyLineChart`
Non technicals folks can also go through the story books.

Reply:
done